src/posteffect/posteffect_posteffectqueue.js

```diff
@@ -226,10 +206,11 @@ pc.extend(pc, function () {
                         var len = effects.length;
                         if (len) {
                             camera.renderTarget = effects[0].inputTarget;
-                            this._setDepthTarget(this.depthTarget);
+                            this.depthTarget = this.camera.camera._depthTarget;
```

Comment:
Who cares? Look at the original source: this posteffects file CREATED the _depthTarget on camera before, and ForwardRenderer was reading it. So I didn't make it worse.